### PR TITLE
feat: add select connection type step to init command

### PR DESCRIPTION
**Pull request by mshaaban0**
## Summary

Add connection type selector to init command

## Screenshots (if appropriate):

Screenshot 2022-09-06 at 15 11 28

### CODE REVIEW

**Review by massao — APPROVED** [review]

File: lib/cmds/init/success.ts [hunk 1]
```diff
@@ -61,8 +62,5 @@ export default async function success({
     )
   }
 
-  console.log(
-    'Do you want to give us feedback on the experience of using this command?'
-  )
-  console.log(`Share your thoughts at ${FEEDBACK_LINK}`)
+  console.log(`Share your feedback at ${FEEDBACK_LINK}`)
```
Comment (denkristoffer) [inline lib/cmds/init/success.ts:65]: I think it could be clearer this is feedback on the CLI init command only.

Comment (mshaaban0) [reply lib/cmds/init/success.ts:65]: This should be for the CLI as a whole, we need less noise with the code examples :)

Comment (denkristoffer) [reply lib/cmds/init/success.ts:65]: I agree with that.

Comment (denkristoffer) [reply lib/cmds/init/success.ts:65]:
```suggestion
  console.log(`Share your feedback for this CLI at ${FEEDBACK_LINK}`)
```
Maybe?

Comment (mshaaban0) [reply lib/cmds/init/success.ts:65]: Isn't the context quite obvious though?

Comment (denkristoffer) [reply lib/cmds/init/success.ts:65]: Maybe I'm dumb but I don't think it is to me 😆

Comment (mshaaban0) [reply lib/cmds/init/success.ts:65]: 😊 the whole flow will be reviewed by content writers anyway. Feel free to suggest a commit

**Pull request merged**